pkg/platform/version.go

```diff
@@ -61,15 +61,19 @@ func LatestCompatibleVersion(ctx context.Context) (string, error) {
 			return semver.Version{}, false
 		}
 
+		// skip the pre-releases (alpha, rc etc.), in semVer stable releases don't contain "-"
+		if strings.Contains(ghVersion.String(), "-") {
```

Comment:
Would be better to use the library: 
```
	v, _ := semver.Make("4.2.1-alpha.0+other")
	if len(v.Pre) > 0 {
	    fmt.Println(len(v.Pre))
	    fmt.Println(v.Pre)
	}
```
output
```
2
[alpha 0]
```

Reply:
Good suggestion, thanks! `semver.Make()` is an alias of `semver.Parse()` which is already being used one above so I utilised that.